themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_breadcrumbs.html

```diff
@@ -1,3 +1,4 @@
+{% if 0 == 1 %}
```

Comment:
The liquid has a `{% comment %}` and `{% endcomment %}` tags that do something similar and seem to be what's used commonly.

Reply:
https://nationbuilder.com/liquid_basics